apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -114,35 +114,38 @@ def link
       )
     end
 
-    # Cluster id the batch connect app uses
-    # @return [String, nil] cluster id used by app
-    def cluster_id
-      form_config.fetch(:cluster, nil)
+    # The clusters the batch connect app is configured to use
+    # @return [Array<String>, []] the clusters the app wants to use
+    def configured_clusters
+      Array.wrap(form_config.fetch(:cluster, nil)).compact.map(&:to_sym)
```

Comment:
If form doesn't specify cluster, lets default "configured" to all clusters not hidden. At OSC that would be Owens, Pitzer and Ruby

Reply:
Or maybe if form specifies `cluster: *` we do this (all clusters not hidden)? See new requirement at bottom of this PR. Need to think about and discuss.

Reply:
Yeah if cluster isn't specified in the form, we do nothing form wise. I think we would want to be able to support specifying (or changing) the cluster in the submit.yml as well.

Reply:
I didn't implement `*` (or arbitrary regexes?), I just wanted to start out small and iterate.   This thread is a little back and forth, where'd you land on it?

Reply:
You are right to start out small. We can merge and discuss. I think we will want to support the case of omitting it (and thus specifying it in the submit.yml, for example, or via some custom control or a mix of control and form.js). So I think that `cluster: *` would be cool. However, I'm unaware of something that does the equivalent of `Array#grep` in Ruby but with shell expansion instead of regular expressions.